edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java

```diff
@@ -60,6 +65,12 @@
 
 	private LinkedList<Float> boundary;
 
+	/**
+	 * A set of keywords in log messages.
+	 */
+	private final Set<String> KEYWORDS_IN_LOG_MESSAGES = Stream.of("fail", "disable", "error", "exception")
```

Comment:
Add "collision", "reboot", "terminate", "throw", "shut", "start", "shut", and "run".